packages/react-reconciler/src/ReactFiberScheduler.js

```diff
@@ -937,6 +982,18 @@ export default function<T, P, I, TI, HI, PI, C, CC, CX, PL>(
       resetStack();
```

Comment:
Why do we reset all these when we're just going to set them right after here anyway?

Reply:
It's to unwind the stack. I could refactor this a bit but I'd prefer not to in this PR since it's, ahem, perpendicular :D